### PR TITLE
Simplify the quick start process

### PR DESCRIPTION
**Pull request by tcodehuber**
As titled.

### CODE REVIEW

**Comment by tcodehuber** [comment]
WX20240518-232113@2x

**Review by zhoujinsong — APPROVED** [review]
LGTM.

Thanks for the contribution! @tcodehuber

File: amoro-site/content/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+Then you can switch the tab to `Optimizers`, you can find the newly launched optimizer here.
+
+![Optimizers](../images/quickstart/optimizers.png)
+
+{{< hint info >}}
+You may need to wait for up to 30 seconds for the optimizer to register with AMS.
+{{< /hint >}}
+
+## Demo setps
```
Comment (tcodehuber) [inline amoro-site/content/quick-start.md:127]: Demo steps

Comment (zhoujinsong) [reply amoro-site/content/quick-start.md:127]: We can create another PR to fix it.

**Pull request merged**